docs/webdriver-bidi.md

```diff
@@ -0,0 +1,117 @@
+# Experimental WebDriver BiDi Support
+
+[WebDriver BiDi](https://w3c.github.io/webdriver-bidi/) is a new cross-browser
+automation protocol that extends WebDriver with events. Puppeteer aims at
+supporting WebDriver BiDi whenever possible and currently has experimental
```

Comment:
@mathiasbynens on the wording - Based on the current % features supported, is "Experimental" or "Beta" a more suitable word?

From the [definition in wikipedia](https://en.wikipedia.org/wiki/Software_release_life_cycle#:~:text=finally%20gold%20status.-,Beta,-%5Bedit%5D) - 
A beta phase generally begins when the software is **feature-complete** but likely to contain several known or unknown bugs.

Reply:
“Experimental” or “Alpha” seems appropriate. One potential issue is that it’s technically been “experimentally implemented” for a few months already, but what’s new now is that we’ve reached a significant milestone of being able to deprecate the CDP-based Firefox support. I’m not sure if we need a different term for that though — maybe just sticking to “experimental” in combination with this announcement is simplest.

Reply:
SGTM. Let's stick to "experimental".